mibidata/color.py

```diff
@@ -224,3 +225,114 @@ def composite(image, color_map, gamma=1/3, min_scaling=10):
             gamma)
     screened = _screen(data_map)
     return np.uint8(screened * 255)
+
+
+def compose_overlay(image, overlay_settings):
+    """Overlays multiple image channels using overlay_settings from mibitracker.
+
+    Args:
+        image: A MibiImage.
+        overlay_settings: Dictionary in one of the forms:
+            {
+                'image_id': {
+                        'channels': {
+                            'channel1': {'color': color, ...},
+                            'channel2': {'color': color, ...},
+                            ...
+                    }
+                }
+            },
+            {
+                'channels': {
+                        'channel1': {color': color, ...},
+                        'channel2': {color': color, ...},
+                        ...
+                }
+            }, or
+            {
+                'channel1': {'color': color, ...},
+                'channel2': {'color': color, ...},
+                ...
+            }
+            Each channel should have the following fields:
+                'color': One of the following: 'Cyan', 'Yellow', 'Magenta',
+                    'Green', 'Orange', 'Violet', 'Red', 'Blue', or 'Gray'.
+                'brightness': (optional) float between -1 and 1; defaults to 0.
+                'intensity_higher': (optional) Upper limit of the channel
+                    intensity; defaults to maximum counts in the channel.
+                'intensity_lower': (optional) Lower limit of the channel
+                    intensity; defaults to 0,
+                'blur': (optional) float between 0 and 1 defining the gaussian
+                    blur of the channel; defaults to 0.
```

Comment:
The formatting of this doesn't work in the sphinx docs, and I think it can be simplified to reference what one gets from mibitracker rather than reproducing it in its entirety. If you do want to use what you write here in the error message below, save it as a string rather than copy-pasting.

Here's what it looks like now:
![image](https://user-images.githubusercontent.com/1189568/68974657-1cf4eb00-07a6-11ea-9178-60f84334c98e.png)